architecture/secure_storage.rst

```diff
@@ -341,14 +341,22 @@ on the device: this is a one time action for the lifetime of the device. Once
 the key has been written on the eMMC controller, the controller uses it to
 authenticate requests.
 
-This key can be programmed in a number of ways. If you want OP-TEE to do it,
+The RPMB key can be programmed in a number of ways. The safest and most
+secure way is to program it from normal world during production in a secure
+environment. MMC-tools_ can for instance be used to program the RPMB
```

Comment:
Misses inverse quotes around `MMC-tools_`?

Reply:
Why should we have that? We don't use it with mmc-utils_.